packages/app/tests/integration/authentication.test.ts

```diff
@@ -1,96 +1,247 @@
+    const authPage = auth.page;
 
-    await authPage.waitFor(authPageObject.$textareaReadOnlySeedPhrase);
+    await authPage.waitFor(auth.$textareaReadOnlySeedPhrase);
 
-    const $secretKeyEl = await authPage.$(authPageObject.$textareaReadOnlySeedPhrase);
+    const $secretKeyEl = await authPage.$(auth.$textareaReadOnlySeedPhrase);
     if (!$secretKeyEl) {
       throw 'Could not find secret key field';
     }
-    const secretKey: string = await authPage.evaluate(el => el.value, $secretKeyEl);
+    const secretKey = (await authPage.evaluate(el => el.textContent, $secretKeyEl)) as string;
     expect(secretKey.split(' ').length).toEqual(SEED_PHRASE_LENGTH);
     expect(validateMnemonic(secretKey)).toBeTruthy();
 
-    await authPage.click(authPageObject.$buttonCopySecretKey);
+    await authPage.click(auth.$buttonCopySecretKey);
 
-    await authPage.waitFor(authPageObject.$buttonHasSavedSeedPhrase);
-    await authPage.click(authPageObject.$buttonHasSavedSeedPhrase);
+    await authPage.waitFor(auth.$buttonHasSavedSeedPhrase);
+    await authPage.click(auth.$buttonHasSavedSeedPhrase);
 
-    const $usernameInputElement = await authPage.$(authPageObject.$inputUsername);
+    const $usernameInputElement = await authPage.$(auth.$inputUsername);
     if (!$usernameInputElement) {
       throw 'Could not find username field';
     }
     await authPage.type(
-      authPageObject.$inputUsername,
+      auth.$inputUsername,
       `${getRandomWord()}_${getRandomWord()}_${getRandomWord()}_${getRandomWord()}`
     );
-    await authPage.click(authPageObject.$buttonUsernameContinue);
+    await authPage.click(auth.$buttonUsernameContinue);
 
-    await page.waitFor('#auth-response');
-    const authResponseEl = await page.$('#auth-response');
-    const authResponse: string = await page.evaluate(el => el.innerText, authResponseEl);
+    const authResponse = await demoPage.waitForAuthResponse();
     expect(authResponse).toBeTruthy();
-    done();
   }, 120_000);
 
-  // eslint-disable-next-line jest/no-disabled-tests
-  test.skip('signing in with an existing Secret Key', async () => {
-    const { authPage } = await bootstrapConnectModalPageTest(demoPageObject, authPageObject);
+  it('creating a successful local account', async () => {
+    await demoPage.openConnect();
+    await demoPage.clickConnectGetStarted();
+    const authPage = await AuthPage.getAuthPage(browser);
+    const secretKey = await authPage.saveSecretPhrase();
+    expect(secretKey.split(' ').length).toEqual(SEED_PHRASE_LENGTH);
+    expect(validateMnemonic(secretKey)).toBeTruthy();
+    await authPage.clickIHaveSavedIt();
+    await authPage.setUserName(`${getRandomWord()}_${getRandomWord()}_${getRandomWord()}_${getRandomWord()}`);
+    await authPage.page.click(authPage.$buttonUsernameContinue);
+    const authResponse = await demoPage.waitForAuthResponse();
+    expect(authResponse).toBeTruthy();
+  }, 90000);
+
+  it('creating an account - negative scenarious', async () => {
```

Comment:
:+1: for readability, but 👎 for the time it takes to run the tests. Probably worth it for the extra ~30 seconds?

Reply:
If the time becomes a problem we could look into parallelising the tests? There's also the (small) risk that one test passes/fails because of the behaviour of an earlier test.

Reply:
We should be able to run the test in parallel, they are not build on each other and can all run individually.